app/scripts/controllers/network/provider-api-tests/helpers.js

```diff
@@ -177,24 +143,38 @@ function mockRpcCall({ nockScope, request, response, error, delay, times }) {
   // eth-query always passes `params`, so even if we don't supply this property,
   // for consistency with makeRpcCall, assume that the `body` contains it
   const { method, params = [], ...rest } = request;
-  const httpStatus = response?.httpStatus ?? 200;
-  let completeResponse;
+  let httpStatus = 200;
+  let completeResponse = { id: 2, jsonrpc: '2.0' };
```

Comment:
Hmm, why default to an id of `2` here?

Reply:
Oops, I just now noticed this. I suspect this was copied from the core tests, although, I am not sure. I will look into this more tomorrow.

Reply:
Okay, I was right, this appears to have come from the changes that Zach made. As to why exactly 2 was chosen, I am not sure. Perhaps to stand out.